src/App.js

```diff
@@ -64,6 +114,19 @@ class App extends React.Component {
       });
     } else { this.setState({ mounted: true }); }
   };
+  componentDidUpdate = (prevProps) => {
+    if (prevProps.newUser !== this.props.newUser) {
```

Comment:
Sorry I don't understand, Why do we check if they're opposites? Is it because the initial state of `prevProps.newUser` is set to False and then we check when it's true? Is it possible for previous state to be true and then next to be FAlse (eg. they use the app long enough that the token expires and the register call is made again), which would trigger the tutorial again?